### PR TITLE
[Example] PPO simplified example

### PR DESCRIPTION
**Pull request by albertbou92**
## Description

Simplified PPO code example.

## Motivation and Context

Why is this change required? What problem does it solve?
If it fixes an open issue, please link to the issue here.
You can use the syntax `close #15213` if this solves the issue #15213

- [ ] I have raised an issue to propose this change ([required](https://github.com/pytorch/rl/issues) for new features and bug fixes)

## Types of changes

What types of changes does your code introduce? Remove all that do not apply:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds core functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)
- [x] Example (update in the folder of examples)

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [x] I have read the [CONTRIBUTION](https://github.com/pytorch/rl/blob/main/CONTRIBUTING.md) guide (**required**)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: examples/ppo/config2.yaml
Comment (vmoens) [inline examples/ppo/config2.yaml:1]: what is config2? Do we have better config names?

**Review by vmoens — APPROVED** [review]
Superb!
If you have some training results, like an image of training curve or smth, maybe we could put that in this folder in a .md file? Wdyt?

File: examples/ppo/config2.yaml
Comment (albertbou92) [reply examples/ppo/config2.yaml:1]: Yes I have results! I can totally add them. 

Regarding config2, I was using as a reference for testing the original PPO paper (https://arxiv.org/abs/1707.06347), so I tested the config for one Atari environment and one MuJoCo environment. config2 is the configuration for HalfCheetah with the hyperparameters of the original paper.

**Review by vmoens — COMMENTED** [review]
another thing: let's maybe move the imports in the main function?
Like that we can use hydra + submitit/slurm to launch jobs
The idea is that from a jump node you do
python ppo.py blabla
and submitit starts the jobs for you
but for that we need to have a script with minimal imports (you never know how brittle your jump node can be)

**Comment by vmoens** [comment]
I also see that the examples CI are failing, let's fix that

**Comment by albertbou92** [comment]
I think it should be fine now! The remaining tests failing seem to be due to other code

**Pull request merged**